cvat/apps/engine/backup.py

```diff
@@ -1081,21 +1087,26 @@ def _import(importer, request, queue, rq_id, Serializer, file_field_name, locati
                 failure_ttl=settings.IMPORT_CACHE_FAILED_TTL.total_seconds()
             )
 
-        rq_job = queue.enqueue_call(
-            func=import_resource_with_clean_up_after,
-            args=(importer, filename, request.user.id, org_id),
-            job_id=rq_id,
-            meta={
-                'tmp_file': filename,
-                **get_rq_job_meta(request=request, db_obj=None)
-            },
-            depends_on=dependent_job,
-            result_ttl=settings.IMPORT_CACHE_SUCCESS_TTL.total_seconds(),
-            failure_ttl=settings.IMPORT_CACHE_FAILED_TTL.total_seconds()
-        )
+        user_id = request.user.id
+        cm = queue.connection.lock(f'{queue.name}-lock-{user_id}', timeout=30) if settings.ONE_RUNNING_JOB_IN_QUEUE_PER_USER else nullcontext()
```

Comment:
Let's move the code into a function. Something like
def get_rq_lock(request):
    ...